config/samples/nnf_v1alpha1_nnfcontainerprofile.yaml

```diff
@@ -37,8 +27,19 @@ data:
           #   mountPath: /foo/persistent
           # - name: nnf-config
           #   mountPath: /nnf/config
-      tolerations:
-        - effect: NoSchedule
-          key: cray.nnf.node
-          operator: Equal
-          value: "true"
+---
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfContainerProfile
+metadata:
+  name: forever
+  namespace: nnf-system
+data:
+  template:
+    spec:
+      containers:
+      - name: forever
+        image: alpine:latest
+        command:
+        - /bin/sh
+        - -c
+        - "while true; do date && sleep 5; done"
```

Comment:
Did you mean to check this in?

Reply:
Yes. It's a good example of a container that never exits. Hence the name 'forever'

Reply:
The container profile should not define the security context or tolerations since those are values populated by Rabbit software.

I will restore the storages.